app/assets/javascripts/components/editor.js

```diff
@@ -38,9 +38,21 @@ var Editor = (function($, autosize) {
          * @type {function[]}
          */
         this.commands = {
-            'text': function($context) {
-                return $context.data().editorMarkupText;
+            text: function($context) {
+                return $context.data('editorMarkupText');
             },
+            sensor: function($context) {
+                var id = $context.data('editor-markup-sensor');
+                var fallback = prompt('Enter a fallback value that will be output in case that there is no recent sensory data for this sensor.');
+
+                return '{ value(' + id + ')'+ (fallback ? ' | "' + fallback + '"' : '') + ' }';
+            },
+            event: function($context) {
+                var id = $context.data('editor-markup-event');
+                var fallback = prompt('Enter a fallback value that will be output in case that there is no recent data for this event.');
+
+                return '{ date(' + id + ')'+ (fallback ? ' | "' + fallback  + '"' : '') + ' }';
```

Comment:
So this will be inserted as soon as sb. enters a sensor? I like the solution. Only concern: Will the extra effort start to annoy me at some point?